recipes/discasm/post-link.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo "DISCASM requires TRINITY_HOME to be set to the install directory of trinity."
+echo "This version of DISCASM will look for Trinity in PATH, if TRINITY_HOME is not set."
```

Comment:
If this is the case, it is how it should be and I would remove this file.

Reply:
ok. I will do that.